requirements.txt

```diff
@@ -2,8 +2,6 @@
 # readthedocs.org don't support pip8.
 # More info at https://github.com/rtfd/readthedocs.org/issues/1843
 
--r docs/requirements.txt
-
```

Comment:
The prod ``requirements.txt`` file should not be needed for rtd to build the docs. According to the configuration in rtd, it's only been using ``docs/requirements.txt`` anyway, and I guess it's been working fine so far. :)